kafka-connect-fitbit-source/src/main/java/org/radarbase/connect/rest/fitbit/user/Users.java

```diff
@@ -19,16 +19,20 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import org.radarbase.connect.rest.fitbit.util.Page;
+
 import java.util.ArrayList;
 import java.util.List;
 
 public class Users {
   private final List<LocalUser> users;
+  private final Page metadata;
 
 
   @JsonCreator
-  public Users(@JsonProperty("users") List<LocalUser> users) {
+  public Users(@JsonProperty("users") List<LocalUser> users, @JsonProperty("metadata") Page metadata) {
     this.users = new ArrayList<>(users);
+    this.metadata = metadata;
```

Comment:
Is the metadata being used anywhere?
If not would it not be better to ignore this? You then also don't need to add the Page class.

Reply:
Made the changes. Also have additional changes: changed `authorized` and fixed the scope.